notice_and_comment/settings/base.py

```diff
@@ -21,4 +21,6 @@
 
 # Commenting
 BROKER_URL = 'redis://localhost:6379/0'
-REGS_API_URL = 'https://api.data.gov/regulations/v3/'
+REGS_API_BASE = 'https://api.data.gov/TEST/regulations/v3/comment'
+REGS_API_URL = '{}?api_key={}'.format(
+    REGS_API_BASE, os.environ.get('REGS_API_KEY', 'DEMO_KEY'))
```

Comment:
Since this issue comes up so often, I have something to handle it in cfenv--`AppEnv.get_credential` returns the requested credential from the first matching service, and if none match, it checks the environment. See https://github.com/jmcarp/py-cfenv/blob/master/cfenv/__init__.py#L62-L70 for details.